src/components/Comments.jsx

```diff
@@ -1,26 +1,54 @@
 import { useEffect, useState } from "react"
-import { getComments } from "../utils/api"
+import { getComments, getSingleReview } from "../utils/api"
 import { useParams, useNavigate} from "react-router-dom"
 
 function Comments () {
     const { review_id } = useParams()
     const [comments, setComments] = useState([])
     const[isLoading, setIsLoading] =useState(true)
     const navigate = useNavigate()
+    const [review, setReview] = useState([])
     
     useEffect(() => {
         setIsLoading(true)
         getComments(review_id).then((data) => {
             setComments(data)
+            setIsLoading(false)
         })
-        setIsLoading(false)
     }, [review_id])
 
+    useEffect(() => {
+        setIsLoading(true)
+        getSingleReview(review_id).then((data) => {
+            setReview(data)
+            setIsLoading(false)
+        })
+    },[review_id])
+
```

Comment:
These two useEffects can be combined to save space and make the code more concise, and there's no reason to have two separate ones in this case